ldcbench.http-node/src/main/java/org/dice_research/ldcbench/nodes/http/simple/dump/DumpFileBuilder.java

```diff
@@ -90,6 +94,9 @@ private OutputStream generateOutputStream()
         OutputStream out = new FileOutputStream(dumpFile);
         out = new BufferedOutputStream(out);
         if (compression != null) {
+            if (compression.getFileNameExtension().equals(".br")) {
+                Brotli4jLoader.ensureAvailability();
+            }
```

Comment:
This `if` statement is misplaced here. It forces the `DumpFileBuilder` to be aware of a function that is very specific to the Brotli compression. This breaks the whole idea of keeping the functionality of the compression hidden from the `DumpFileBuilder`. Please think about a better solution for this.